modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_renderer/page.jsp

```diff
@@ -17,7 +17,7 @@
 <%@ include file="/data_layout_renderer/init.jsp" %>
 
 <div class="sheet">
-	<div class="container-fluid-1280 ddm-form-builder-app">
+	<div className="container-fluid container-fluid-max-xl ddm-form-builder-app">
```

Comment:
Not sure what happened here... maybe the macro failed? I assume this should be 

```html
<clay:container
    className="ddm-form-builder-app"
>
    ...
</clay:container>
```

Reply:
Reverted case with a typo (I was working on this)